Header header
int32 seq_no
string node_name
string node_status
